src/main/java/seedu/duke/command/ViewNoteCommand.java

```diff
@@ -68,8 +72,17 @@ public String execute() {
             }
         }
         if (!noteExists) {
-            return "This note does not exists in the notebook";
+            return COMMAND_UNSUCCESSFUL_MESSAGE;
         }
-        return note.getContent();
+        ArrayList<Tag> notetags = note.getTags();
+
+        // format output string
+        String stringToPrint = note.getTitle() + "\n";
```

Comment:
To standardize with the rest of the new lines
```suggestion
        String stringToPrint = note.getTitle() + InterfaceManager.LS;
```